api/project/models/project.js

```diff
@@ -5,4 +5,112 @@
  * to customize this model
  */
 
-module.exports = {};
+const axios = require('axios');
+const _ = require('lodash');
+
+module.exports = {
+  lifecycles: {
+    async beforeUpdate(params, data) {
+      try {
+        let projectBeforeModification = await strapi
+          .query('project')
+          .findOne(params, ['heroImage', 'heroImage.formats']);
+        let concatinated = [
+          ...projectBeforeModification.openPositions,
+          ...data.openPositions,
+        ]
+          .filter(
+            (v, i, a) =>
+              a.findIndex(
+                (t) =>
+                  t.id === v.id &&
+                  t.title === v.title &&
+                  t.description === v.description &&
+                  t.isHidden === v.isHidden
+              ) === i
+          )
+          .slice(projectBeforeModification.openPositions.length);
+        let deletedFields = _.differenceBy(
+          projectBeforeModification.openPositions,
+          data.openPositions,
+          'id'
+        );
+
+       
+        if (
+          !_.isEqual(
+            projectBeforeModification.openPositions,
+            data.openPositions
+          ) && process.env.NODE_ENV == 'production'
+        ) {
+          await axios.post(
+            process.env.OPEN_POSITIONS_DISCORD_WEBHOOK,
+            {
+              username: `${projectBeforeModification.title} Notifier`,
+              avatar_url:
+                'https://avatars.githubusercontent.com/u/53379976?s=200&v=4',
+              embeds: [
+                {
+                  title: `${projectBeforeModification.title} Project Page`,
+                  url: `https://devlaunchers.org/projects/${projectBeforeModification.slug}`,
+                  description: `*someone* **__UPDATED__** the open roles => [Link to the strapi page](${process.env.FRONTEND_URL}/admin/plugins/content-manager/collectionType/application::project.project/${params.id})`,
+                  color: 15258703,
```

Comment:
> We can make a constant for this color, otherwise I have no idea what color it is.

This is the color of the stripe on the embed shown in left side of the discord message, refer to the screenshot. it's a decimal color value so I'm just going to leave a comment with the color name and telling it's a decimal value.